harambe_scrapers/extractor/wayne_commission/listing.py

```
@@ -0,0 +1,358 @@
+                                "AppleWebKit/537.36 (KHTML, like Gecko) "
+                                "Chrome/138.0.0.0 Safari/537.36 Edg/138.0.0.0"
+                            ),
+                            "x-b3-sampled": "1",
+                            "x-b3-spanid": "b976fbfea423163d",
+                            "x-b3-traceid": "be91badf5d062cf9732c40defae7b6b0",
+                            "x-oracle-apm-ba-version": "1.1.131",
+                            "x-requested-with": "XMLHttpRequest",
+                            "Cookie": "ASP.NET_SessionId=h4ooyutzz33houczonxywo0h",
+                        }
+
+                        response = requests.request(
```

Ruff (0.14.6)

Probable use of `requests` call without timeout

(S113)